common/v2/features/SendAssets/stateFactory.tsx

```diff
@@ -125,7 +126,14 @@ const TxConfigFactory: TUseStateReducerFactory<State> = ({ state, setState }) =>
       .finally(cb);
   };
 
-  const handleSignedTx: TStepAction = (payload: ISignedTx, cb) => {
+  const handleSignedTx: TStepAction = (payload: any, cb) => {
```

Comment:
Please avoid breaking type-safety.
Prefer updating `ISignedTx` or adding an option to the argument `payload: ISignedTx | <whateveryouneed>`

Reply:
Done c: